src/config.toml

```diff
@@ -0,0 +1,36 @@
+[[completion]]
+type="ruby_code"
+trigger="rb (.*)"
+
+[[completion]]
+type="numeric_expression"
+trigger="([0-9]+.*)"
+
+[[completion]]
+type="url"
+trigger="g (.*)"
+command="ruby -r uri -e '@q = \"{}\"; @q_quoted = URI.escape(@q); puts \"Google for #@q\thttps://www.google.com/search?q=#@q_quoted\"'"
+
+[[completion]]
+type="command"
+command="echo {}"
```

Comment:
I'd like to remove that. It causes my completions to only work on the second press of tab...